src/components/__tests__/LeaderboardTable.spec.js

```diff
@@ -0,0 +1,35 @@
+import { describe, it, expect } from "vitest";
+import { mount } from "@vue/test-utils";
+import { createVuetify } from "vuetify";
+import LeaderboardTable from "../LeaderboardTable.vue";
+
+describe("LeaderboardTable", () => {
+  const vuetify = createVuetify();
+  const profileBaseUrl = "https://meta.wikimedia.org/wiki/User:";
+  it("renders properly", () => {
+    const LeaderboardTableProps = {
+      contributions: [
+        {
+          userName: "Hannah Waruguru",
+          toolsEdited: 12,
+        },
+      ],
+      title: "All Time Best",
+      profileBaseUrl,
+    };
+    const wrapper = mount(LeaderboardTable, {
+      props: LeaderboardTableProps,
+      global: {
+        plugins: [vuetify],
+      },
+    });
+    expect(wrapper.text()).toContain("All Time Best");
+
+    const firstRow = wrapper.findAll('td')[0].find('a')
```

Comment:
I think 'td' refers to the columns in the table, so the name of this variable is a bit confusing.

I'd suggest we change it to something like this: 
```
rows = wrapper.find('tbody').findAll('tr');
firstRow = rows[0];

userName = firstRow.find('a');
expect(userName.text()).toBe('Hannah Waruguru');
expect(userName.attributes().href).toBe('https://meta.wikimedia.org/wiki/User:Hannah_Waruguru');

contributionCol = firstRow.findAll('td')[1];
expect(contributionCol.text()).toBe('12');
```

I am yet to test it, this is only a demonstration to explain what I mean.

Reply:
Oh yeah, I missed this.correcting it rn